### PR TITLE
[16.0][ADD] product_packaging_multi_barcode

### PR DESCRIPTION
**Pull request by QuocDuong1306**
This module allows to define multiple barcodes on product packagings.

### CODE REVIEW

File: product_packaging_multi_barcode/models/product_barcode.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    raise ValidationError(
+                        _(
+                            'The Barcode "%(barcode_name)s" already exists for '
+                            'product "%(product_name)s" in the company %(company_name)s'
+                        )
+                        % dict(
+                            barcode_name=record.name,
+                            product_name=product.name,
+                            company_name=product.company_id.name,
+                        )
```
Comment (simahawk) [inline product_packaging_multi_barcode/models/product_barcode.py]:
```suggestion
                    raise ValidationError(
                        _(
                            'The Barcode "%(barcode_name)s" already exists for '
                            'product "%(product_name)s" in the company %(company_name)s',
                            barcode_name=record.name,
                            product_name=product.name,
                            company_name=product.company_id.name,
                        )
```

File: product_packaging_multi_barcode/models/product_barcode.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    raise ValidationError(
+                        _(
+                            'The Barcode "%(barcode_name)s" already exists for '
+                            'packaging "%(name)s" in the company %(company_name)s.'
+                        )
+                        % dict(
+                            barcode_name=record.name,
+                            name=packaging.name,
+                            company_name=packaging.company_id.name or "",
+                        )
```
Comment (simahawk) [inline product_packaging_multi_barcode/models/product_barcode.py]:
```suggestion
                        _(
                            'The Barcode "%(barcode_name)s" already exists for '
                            'packaging "%(name)s" in the company %(company_name)s.',
                            barcode_name=record.name,
                            name=packaging.name,
                            company_name=packaging.company_id.name or "",
                        )
```

**Review by simahawk — COMMENTED** [review]
LG

**Comment by QuocDuong1306** [comment]
Hi @simahawk , I updated, thanks for your suggestions

**Review by simahawk — APPROVED** [review]

**Comment by luke-stdev001** [comment]
> Hi @simahawk , I updated, thanks for your suggestions

@QuocDuong1306 ,

Happy to do a functional review on this as a bystander if it would be helpful.

**Comment by QuocDuong1306** [comment]
Of course, Thank you @luke-stdev001

**Review by santostelmo — APPROVED** [review]

**Pull request merged**